backend/README.md

```diff
@@ -103,3 +103,11 @@ Note however that this results in a) all history being saved in memory instead o
 
 We use [state migrations](./src/database/state-migrations/) to convert outdated states to new versions.
 Look at [`migrations.ts`](./src/database/state-migrations/migrations.ts) for more information.
+
+### Note on long term storage
+
+The current setup when using a database is that no exercises get deleted unless anyone deletes them from the UI (or, to be more precisely, using the HTTP request `DELETE /api/exercises/:exerciseId`).
+An average exercise with four sections being actively played and about 45 minutes of exercise time seems to take up about 10 MB storage in the database.
+This is in itself not much but in case many exercises of this size are run each day, it can scale quickly.
+Also note that there can never be more than 10,000 exercises because then the id generator fails.
+Finally note that it may be an issue of data protection laws like the [GDPR](https://gdpr.eu/) when user data never gets deleted, even after years of not using it.
```

Comment:
```suggestion
The current setup when using a database is that no exercises get deleted unless anyone deletes them from the UI (or, more precisely, using the HTTP request `DELETE /api/exercises/:exerciseId`).
An average exercise with four sections being actively played and about 45 minutes of exercise time seems to take up about 10 MB of storage in the database.
This is not much in itself, but if many exercises of this size are run each day, it can scale quickly.
Also, note that there can never be more than 10,000 exercises because then the id generator fails.
Finally, note that it may be an issue of data protection laws like the [GDPR](https://gdpr.eu/) when user data never gets deleted, even after years of not using it.
```

Reply:
-> from Grammarly